main.rb

```diff
@@ -1,7 +1,8 @@
 require './app'
 
+puts 'Welcome to School Library App!'
+
 def display_menu
-  puts 'Welcome to School Library App!'
   puts 'What would you like to do?'
   puts '1. List all books'
   puts '2. List all people'
```

Comment:
- [ ] Everything looks great just please add ``parent_permission: true`` to the student class and teacher class in order to be seen as a default
```ruby
class Student < Person 

  def initialize(age, classroom, name = "Unknown", parent_permission: true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
done